frontend/app/helpers/spree/navigation_helper.rb

```diff
@@ -57,7 +57,7 @@ def spree_menu(location = 'header')
       method_name = "for_#{location}"
 
       if available_menus.respond_to?(method_name) && Spree::Menu::MENU_LOCATIONS_PARAMETERIZED.include?(location)
-        available_menus.send(method_name, I18n.locale, current_store.default_locale)
+        available_menus.send(method_name, current_store)
```

Comment:
@MatthewKennedy we still should pass the locale here so if we have multi-language store we should show the menu with this locale or fallback to the default one

Reply:
@damianlegawiec I moved `I18n.locale` to the receiving end of this method in the Menu model, is that not advised?